lib/matrix.h

```diff
@@ -198,6 +207,20 @@ inline void Matrix4Permute(Array2& matrix) {
   std::swap(matrix[26], matrix[28]); std::swap(matrix[27], matrix[29]);
 }
 
+// Dagger a 4x4 matrix.
+template <typename Array2>
+inline void Matrix4Dagger(Array2& matrix) {
+  for(int i = 1;i < 32;i+=2) {
```

Comment:
Fix spaces and use unsigned for consistency: `for (unsigned i = 1; i < 32; i += 2) {`.